apps/files/src/components/FileEntry/FileEntryPreview.vue

```diff
@@ -50,6 +50,10 @@
 			:aria-label="t('files', 'Favorite')">
 			<FavoriteIcon v-once />
 		</span>
+
+		<OverlayIcon :is="fileOverlay"
```

Comment:
This will break once we migrate to vue3 so you can already prepare by using `<component>` instead:
```suggestion
		<component :is="fileOverlay"
```